sdk/communication/communication-identity/samples/v1/javascript/createUserAndToken.js

```diff
@@ -18,14 +18,19 @@ async function main() {
   console.log("\n== Create User and Token Sample ==\n");
   const client = new CommunicationIdentityClient(connectionString);
   const scopes = ["chat"];
+  const customId = "alice@contoso.com";
 
   // Create user with default token
   console.log("Creating User and Token");
-  const communicationUserToken = await client.createUserAndToken(scopes);
+  const communicationUserToken = await client.createUserAndToken(scopes, { customId: customId });
```

Comment:
In JS there's a shorthand for when property name and value name are the same.

```suggestion
  const communicationUserToken = await client.createUserAndToken(scopes, { customId });
```